crates/util/python-utils/src/error.rs

```diff
@@ -50,6 +53,9 @@ impl From<Error> for PyErr {
             Error::BitcoinAddress => {
                 PyErr::new::<PyTypeError, _>("Not a valid bitcoin address".to_owned())
             }
+            Error::OpReturnTooLong => {
+                PyErr::new::<PyTypeError, _>("OP_RETURN bigger than 80 bytes".to_owned())
```

Comment:
There's a lot of repetitive structure here.  Can we rework this so the match arms all return the `&'static str`s and *then* pass it to the `PyErr` constructor only once?

Also does `TypeError` make sense here?  I think `ValueError` is the right thing to use.  Although maybe this doesn't mean what I think it means.